src/main/java/brandkon/category/CategoryService.java

```diff
@@ -0,0 +1,25 @@
+package brandkon.category;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    public CategoryService(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    public List<CategoryResponse> readAll() {
+        List<Category> allCategory = categoryRepository.findAll();
+        return allCategory.stream().map(category -> new CategoryResponse(
+                category.getId(),
+                category.getName(),
+                category.getSlug(),
+                category.getImageUrl()))
+                .toList();
```

Comment:
이런 경우 중간 변수를 안 만들고 한 줄로 작성해도 좋아요
```suggestion
        return categoryRepository.findAll()
                .stream()
                .map(category -> new CategoryResponse(
                        category.getId(),
                        category.getName(),
                        category.getSlug(),
                        category.getImageUrl()))
                .toList();
```